website/docs/setup/restore-options.md

```diff
@@ -108,3 +108,18 @@ the copy of`reports.txt` is named `reports 1.txt`.
 Collisions will entirely replace the current version of the item with the backup
 version. If multiple existing items collide with the backup item, only one of the
 existing items is replaced.
+
+## To resource
+
+The `--to-resource` flag lets you select which resource will receive the restored data.
+A resource can be a mailbox, user, or sharepoint site.  
+
+<CodeBlock language="bash">{
+    `corso restore onedrive --backup abcd --to-resource adelev@alcion.ai`
```

Comment:
Suggest changing `abcd` to a uuid format instead

Reply:
Going to keep this, as it currently matches our other examples.

Reply:
The structure of this page is a little off beyond just this flag.
* The Item collision handling sub options probably should move to H3 `###` under `## Item collision handling`
* The heading `To resource` while describing the flag correctly is probably not descriptive enough for a heading. Better option is probably `Restore resource target`. For consistency you could rename `Destination` above to `Restore folder target` or `Restore folder destination`
* Probably need to cover the interactions of `to-resource` with `destination`, if any. Assuming that the example restores to an alternate resource in a folder `Corso_Restore_<>` this should probably be mentioned for clarity. If we support combing the parameters, an extra example can be useful as well

Not sure, if related to the functionality here, but some of the behavior around restoring to a resource that may have been deleted at some point for `original location` should be covered somewhere in the docs.

Reply:
fixed the suggested comments.

> some of the behavior around restoring to a resource that may have been deleted at some point for original location should be covered somewhere in the docs

There's the `limitation` at the bottom which states we won't restore to a missing or deleted resource.  Is that sufficient?